frontend/src/ProfilePageComp/SavedRecordings.jsx

```diff
@@ -1,9 +1,11 @@
+/* eslint-disable react/prop-types */
+
 import { useContext, useEffect, useState } from "react";
 import { UserContext } from "../UserContext.js";
 
-function SavedRecordings() {
+function SavedRecordings({ recordings, setRecordings }) {
```

Comment:
I think you'd probably want to have your own state variable for this component to store the fetched list of user recordings. Setting this state variable will update recordings in other parts of the app, unless this was your intention.